wondrous-app/components/Settings/generalSettings.tsx

```diff
@@ -361,7 +407,24 @@ export const PodGeneralSettings = () => {
       },
     });
   }
-
+  const handleArchivePodClick = async () => {
+    await apollo.mutate({
+      mutation: ARCHIVE_POD,
+      variables: {
+        podId,
+      },
+    });
+    router.reload()
```

Comment:
not eactly the right thing todo but since archiving is a rare action this is ok for now